lib/bn.js

```diff
@@ -327,6 +327,13 @@
     dest.red = this.red;
   };
 
+  BN.prototype.move = function move (dest) {
```

Comment:
Oh, shouldn't it be private? It looks like this could lead to funny state issues if used as a part of API.

Reply:
I'll not mind of making it private. Do you want see it as separate function or as "inline" function?

Reply:
IMO, it makes sense to have it on a prototype. Just prefix it with `_`. @dcousens what do you think?

Reply:
@indutny I think the public interface semantics of `move` can be assumed that the originating object is reset or garbage after use.

Reply:
@fanatid prefix it with `_`?

Reply:
@dcousens added prefix `_`, let me know if I should squash commits